pkg/util/k8sutil/pods.go

```diff
@@ -329,6 +330,32 @@ func arangosyncContainer(image string, imagePullPolicy v1.PullPolicy, args []str
 	return c
 }
 
+func arangodbexporterContainer(image string, imagePullPolicy v1.PullPolicy, args []string, env map[string]EnvValue, livenessProbe *HTTPProbeConfig) v1.Container {
+	// THIS IS WORK IN PROGRESS
+	// IN THE END THE ARANGO DOCKERIMAGE WILL CONTAIN THE EXPORTER
+	c := v1.Container{
+		Command:         append([]string{"/app/arangodb-exporter"}, args...),
+		Name:            ExporterContainerName,
+		Image:           "arangodb/arangodb-exporter:0.1.3",
```

Comment:
Suggestion: Make the image configurable in the spec.
If not specified, use the standard arangodb image (as soon as it contains the exporter)